+euphonic/mp_grid.m

```diff
@@ -1,5 +1,22 @@
-function out = mp_grid(sz)
-% Returns a Monkhorst-Pack grid of specified size
-    euphonic_on();
-    out = py.euphonic.util.mp_grid(int32(sz));
-end
+classdef mp_grid < light_python_wrapper.light_python_wrapper
```

Comment:
Is there a reason this was changed from a function to a class? It seems like it might be unnecessarily complicated...

If it is to stay as a class the docstring needs to be updated:
```
To obtain help on this class and its methods please type:
    %
    %   >> import euphonic.doc
    %   >> doc euphonic.mp_grid
```
etc.

Reply:
Ah... yeah, this was code to allow `help(euphonic.mp_grid)` (using the overloaded `help` method of the `light_python_wrapper` class. This could be reverted back if we're not really using that syntax any more...

Reply:
Yeah I think that would be simplest!

Reply:
Umm... actually, the `help euphonic.mp_grid` syntax doesn't work if `mp_grid` is just a function. This is because in order decide which help text (Matlab or Python) to display, it tries to detect if the topic asked for refers to a wrapped object (display Python help) or not (display Matlab help). So, if `mp_grid` is a class and subclasses `light_python_wrapper` it detects it as a wrapped Python object and displays the Python help. But if `mp_grid` is a simple Matlab function, it just returns the Matlab docstring.

I could modify the help code to check if the topic string could refer to a Python module which is loadable and then check if it has the correct attribute [e.g. check if `euphonic` is a loadable module and if it has attribute `mp_grid`]... in this case `help numpy.random.rand` etc would also work in Matlab. However, in our particular case this will not work because mp_grid is a function in the `euphonic.utils` module not `euphonic` directly...

Reply:
Ah ok, I quess it's not that complicated, and serves as a good template for importing other functions. Leave as-is then!